### ISSUE TITLE
Player need double jump when Acrobatics is activated.

### ISSUE THREAD

**Issue by broken1arrow** [mcMMO-Dev/mcMMO#4314]
When player fall to the ground Acrobatics activated and not stop "rolling" (I have similar issue ass admin, but it stop pretty soon), for it shall stop he need jump up. 

I cant turn off ExploitFix, because he get xp all the time.

Plugins is cmi and MCmmo as work in 1.15.2 fine but not in 1.16.1 and still problems in 1.16.2. In 1.16.1 I has same issue as the player get, but not in 1.16.2.

Use version 2.1.149 Mcmmo and cmi is 8.7.9.3. Player has perms for flyc from cmi.

**Comment by nossr50** [comment]
Double Jump? I'm not sure what this report is trying to say.

**Comment by broken1arrow** [comment]
> Double Jump? I'm not sure what this report is trying to say.


sry, meant after Acrobatics get activated. He need to jump one time to turn it off

**Comment by broken1arrow** [comment]
> Double Jump? I'm not sure what this report is trying to say.

My only options I find is. 

Remove/Disable Acrobatics skill
Disable all xp on Acrobatics skill
Downgrade to 1.15.2 when it working
Hope you can fixt, if not I force to Remove plugin (players complaining for long time bout this).

**Comment by nossr50** [comment]
I don't understand what the issue is, could you rephrase it?

**Comment by broken1arrow** [comment]
> I don't understand what the issue is, could you rephrase it?

If I or most players get this bug. When player fall (over 2/3 blocks) Acrobatics activated and not stop only keep going when player walk around or run (he need jump for it turn off).

I get this error in 1.16.1, but not soo common i get the bug in 1.16.2. Only players get it all the time.

**Comment by nossr50** [comment]
Can you replicate this bug if you only have mcMMO running and I mean literally only mcMMO?

**Comment by broken1arrow** [comment]
> Can you replicate this bug if you only have mcMMO running and I mean literally only mcMMO?

I try it out and something trigger ExploitFix if you take dmg when you have flyc on (normally if you use fly in minecraft you not take dmg).  

So I have one more way to fix this (remove fall dmg if player use flyc). 

I most have some strange bypass, because I a´m admin and not get this problem and not get flooded with exp ether. because if I turn of  ExploitFix, players get tons of exp when Acrobatics activate and walking.

**Comment by nossr50** [comment]
This sounds like a plugin compatibility issue and not the fault of mcMMO.

**Comment by broken1arrow** [comment]
> This sounds like a plugin compatibility issue and not the fault of mcMMO.

Yes it something some happen when you use fly and has dmg on (when you land you take dmg and your plugin not detect it properly).  

How has you set the listener priority?

**Comment by nossr50** [comment]
@broken1arrow The issue is whoever wrote your fly command is not cancelling or setting the damage event for fall damage to 0.

**Comment by broken1arrow** [comment]
> @broken1arrow The issue is whoever wrote your fly command is not cancelling or setting the damage event for fall damage to 0.

It should dmg players some a penalty. But i force to turn it of now.
